app/scripts/modules/core/authentication/userMenu/userRolesModal.html

```diff
@@ -0,0 +1,18 @@
+<div modal-page class="flex-fill">
+  <modal-close dismiss="$dismiss()"></modal-close>
+  <div class="modal-header">
+    <h3>Roles</h3>
+  </div>
+  <div class="modal-body flex-fill">
+    <div ng-if="!ctrl.userRoles.length" >
+      No roles
+    </div>
+    <div ng-repeat="role in ctrl.userRoles | orderBy:'toString()'">
```

Comment:
Not sure why you need `toString()` here, since the values are already strings. You can supply '+' or '-' for ascending/descending.